google/cloud/storage/examples/storage_default_object_acl_samples.cc

```diff
@@ -64,17 +41,11 @@ void ListDefaultObjectAcl(google::cloud::storage::Client client, int& argc,
     }
   }
   //! [list default object acl]
-  (std::move(client), bucket_name);
+  (std::move(client), argv[0]);
```

Comment:
We've been moving to `argv.at(0)` to aid debug-ability should `RunAll()` have some mismatch.